Disabled states and data test attribute

cawo88 opened the pull request:
This PR adds the following ... 

- Toggle Switch disabled state
- Tooltip disabled state
-  Select input `data-test` attribute for Cypress test

for improvements as part of [ENG-2989-investment-proposal-page-bottom-bar-and-new-design](https://github.com/LIQIDTechnology/liqid-web-app/pull/885) Web App PR.

### Testing
I tested the ✅ browsers.
*The percentage shows the current makertshare in Germany*

#### Desktop Browsers
```
✅ Chrome - Desktop - 35%
```

#### Breaking changes
:warning: Will have to make a release and update the package version number in the Web App branch.

varghesethomase commented on src/components/Select/Select.js:
```diff
@@ -575,14 +576,15 @@ class Select extends React.Component {
                 onChange={this.onChangeNative}
                 disabled={this.props.disabled}
                 name={this.props.name}
+                dataTest={this.props.dataTest}
```
dataTest has to be added to props

cawo88 commented:
@varghesethomase thanks for the reminder. I added the propType check and also added `data-test` attribute to the tooltip as well. Please check again when you're free, thanks :)

Select propType check [commit](https://github.com/LIQIDTechnology/liqid-react-components/pull/25/commits/cf5b7c1460d0fd9e1c142b859009b30aa0ed5b27)
Tooltip `data-test` attribute [commit](https://github.com/LIQIDTechnology/liqid-react-components/pull/25/commits/13e2b57f6ea3ddcf934f0a8d98cd844f97cffaaf)
